android/KMAPro/kMAPro/src/main/java/com/tavultesoft/kmapro/MainActivity.java

```diff
@@ -267,9 +267,11 @@ protected void onResume() {
 
     // onConfigurationChanged() only triggers when device is rotated while app is in foreground
     // This handles when device is rotated while app is in background
+    // using KMManager.getOrientation() since getConfiguration().orientation is unreliable #10241
```

Comment:
getResources().getconfiguration() was sometimes reporting PORTRAIT when the device was in LANDSCAPE orientation.